tests/test_suites/GaneshaTests/test_nfs_ganesha_client_io_error_on_master_kill.sh

```diff
@@ -0,0 +1,84 @@
+#
+# To run this test you need to add the following lines to /etc/sudoers.d/saunafstest:
+#
+# saunafstest ALL = NOPASSWD: /bin/mount, /bin/umount, /bin/pkill, /bin/mkdir, /bin/touch
+# saunafstest ALL = NOPASSWD: /usr/bin/ganesha.nfsd
+#
+# The path for the Ganesha daemon should match the installation folder inside the test.
+#
+
+timeout_set 1 minute
+
+CHUNKSERVERS=1 \
+	MOUNT_EXTRA_CONFIG="sfscachemode=NEVER" \
+	setup_local_empty_saunafs info
+
+test_error_cleanup() {
+	cd "${TEMP_DIR}"
+	sudo umount -l "${TEMP_DIR}/mnt/ganesha"
+	sudo pkill -9 ganesha.nfsd
+}
+
+# Create PID file for Ganesha
+PID_FILE=/var/run/ganesha/ganesha.pid
+if [ ! -f ${PID_FILE} ]; then
+	echo "ganesha.pid doesn't exists, creating it...";
+	sudo mkdir -p /var/run/ganesha;
+	sudo touch "${PID_FILE}";
+else
+	echo "ganesha.pid already exists";
+fi
```

Comment:
Not for this commit, but looks like this part is common for many ganesha tests. My suggestion is to move them to another file and just source it into the tests. For instance, if you move them to a file called ganesha_common.sh, the you can use it like this:

```bash
. "$(pwd)/test_suites/GaneshaTests/ganesha_common.sh"
```
This way, the functions and variables will be available, without repeating.

Reply:
Very good suggestion. As you propose, I will add a PR to refactor this suggestion and also remove not needed commented instructions at the beginning of most of the tests of `GaneshaTests` suite.